include/swift/AST/ASTContext.h

```diff
@@ -865,6 +865,12 @@ class ASTContext {
   
   /// Whether our effective Swift version is in the Swift 3 family
   bool isSwiftVersion3() const { return LangOpts.isSwiftVersion3(); }
+  
+  /// Whether our effective Swift version is in the Swift 4 family
+  bool isSwiftVersion4() const { return LangOpts.isSwiftVersion4(); }
+  
+  /// Whether our effective Swift version is in the Swift 5 family
+  bool isSwiftVersion5() const { return LangOpts.isSwiftVersion5(); }
```

Comment:
Actually, I'd rather we not get `isSwiftVersion5`. We shouldn't ever be checking it, because any new behaviors should be "5 or newer"